guides/common/modules/proc_configuring-dynamic-dns-update-with-tsig-authentication.adoc

```diff
@@ -98,17 +98,29 @@ Normally, {foreman-installer} ensures that the `foreman-proxy` user belongs to t
 .Testing External Updates to the DNS Zone in the IdM Server
 
 ifndef::foreman-deb[]
```

Comment:
@ekohl There's one more occurence of these steps, which is also in master+3.1+3.1. It wasn't reported as problematic in the BZ, but I'm making an assumption that it's the same case potentially. Can you confirm?

Reply:
Looking at this, I took a step back and I think we need to reconsider a few things.

First of all, at least in master the installer guarantees that `bind-utils` is installed even when `--foreman-proxy-dns-managed=false`, at least since https://github.com/theforeman/puppet-foreman_proxy/commit/54067fe5ac742000d3c2cdb8505d8f191383aeb7. That's part of Foreman 2.0. From reading the source, it means we can drop the whole install command, which removes the need for package unlocking altogether.

At least in this guide, because of the above installer command. I haven't looked at the other guide yet.